File: Lab1/Lab1.py
def function1():
    pass
    expr = input("Enter an expression: ")
    val = eval(expr)
    print(expr, "is", val, "with type", type(val).__name__)

def function2():
    pass
    print("  .   ")
    print(" / \  ")
    print("/   \ ")
    print("+---+ ")
    print("|   | ")
    print("+---+ ")

def function3():
    yards = int(input("Enter field length in yards: "))
    print("The field is ", yards * 36,  "inches long.")
if __name__ == "__main__":
    function3()

File: Lab1/lab1.js
const readline = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
})

function function1() {
  readline.question("Enter an expression: ", expr => {
    let val = eval(expr)
    console.log(expr + " is " + val + " with type " + typeof(val))
  })
}

function function2() {
  console.log("  .   ")
  console.log(" / \\  ")
  console.log("/   \\ ")
  console.log("+---+ ")
  console.log("|   | ")
  console.log("+---+ ")
  console.log()
}

function function3() {
  readline.question("Enter the field length in yards: ", yards => {
    console.log("The field is " + (Number(yards) * 36) + " inches long.")
  })
}

// function1()
// function2()
// function3()
